pcdsdevices/utils.py

```diff
@@ -687,3 +690,151 @@ def post_ophyds_to_elog(objs, allow_child=False, hutch_elog=None):
 
     hutch_elog.post(final_post, tags=['ophyd_status'],
                     title='ophyd status report')
+
+
+def reorder_components(
+    cls: type,
```

Comment:
Maybe
```suggestion
    cls: type[Device],
```

Reply:
I realized I was leaving off the `Device` info but wasn't sure the right way to hint it. I'll do this in all the places.